website/static/rss.xslt

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<xsl:template match="/">
+  <html>
+  <head>
+    <title>Blog Information</title>
+    <link rel="stylesheet" type="text/css" href="/styles.css" />
+  </head>
+  <body>
+  <main class="markdown">
+    <div class="description">
+    <h1><xsl:value-of select="rss/channel/title"/></h1>
+    <p>Description: <xsl:value-of select="rss/channel/description"/></p>
+    <p>Last Build Date: <xsl:value-of select="substring(rss/channel/lastBuildDate,0,11)"/></p>
+    <p>Language: <xsl:value-of select="rss/channel/language"/></p>
+    </div>
+    <h2>Recent Posts</h2>
+    <ul class="postsList">
+      <xsl:for-each select="rss/channel/item">
+        <li>
+          <a href="{link}">
+            <xsl:value-of select="title"/>
+          </a>
+        </li>
+      </xsl:for-each>
+    </ul>
+    </main>
+  </body>
+  </html>
+</xsl:template>
+
+</xsl:stylesheet>
```

Comment:
This file can't be in `website/static`: this would only work for our own website. We need to put these things in the blog plugin itself and let it copy the default files at the end of the build, during the feed generation process.

The plugin must ensure that the referenced stylesheet in our feed always exists, and we should also use a name that is not likely to conflict with existing static files. Maybe sites using 2 times the blog plugin would want to provide different XSLT and CSS files for each blog.

If you want a concrete example:
- http://thisweekinreact.com/articles
- http://thisweekinreact.com/newsletter

What if I want to render these 2 feeds differently?
- https://thisweekinreact.com/articles/rss.xml
- https://thisweekinreact.com/newsletter/rss.xml